src/client/app/components/Movie/Movie.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import styles from './Movie.less';
+
+function Movie(props) {
```

Comment:
You can try to use destructing here in prop argument, and then use it once again for properties of `movie` object